contractcourt/channel_arbitrator.go

```diff
@@ -1351,6 +1340,28 @@ func (c *ChannelArbitrator) sweepAnchors(anchors *lnwallet.AnchorResolutions,
 			},
 		)
 
+		// Define a deadline height that's default to none.
+		deadlineHeight := fn.None[int32]()
+		deadlineDesc := "None"
+
+		// If we have a deadline, we'll use it to calculate the
+		// deadline height.
+		deadline.WhenSome(func(d int32) {
+			deadlineHeight = fn.Some(d + int32(heightHint))
+			deadlineDesc = fmt.Sprintf("%d", d)
+		})
+
+		// Calculate the budget.
+		budget := calculateBudget(
+			value, c.cfg.Budget.AnchorCPFPRatio,
+			c.cfg.Budget.AnchorCPFP,
+		)
```

Comment:
Updated the docs - think I'll create a detailed README here in 0.18.1.

Reply:
then what if you don't use that much as fees? Say you have an HTLC of 1 btc, then you allocate 0.01 btc as fees when sweeping it, the rest 0.99 btc will be the value left, if you set `AnchorCPFPRatio` to 0.1, then this would be 0.099 btc.

Or do you mean use the `AnchorCPFPRatio` and `AnchorCPFP` directly?